rancid-tomatillos/src/components/Login.js

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import {BrowserRouter as Router, Route} from "react-router-dom";
+import MainView from "./MainView";
+
+class Login extends React.Component {
+    constructor() {
+        super();
+        this.state = {
+            email: "",
+            password: "",
+        };
+    }
+    updateValue = (event) => {
+        this.setState({[event.target.name]: event.target.value});
+    };
+    submitUserInfo = (event) => {
+        event.preventDefault();
+        this.postUserData();
+    };
+
+    async postUserData() {
+        const promise = await fetch(
+            "https://rancid-tomatillos.herokuapp.com/api/v2/login",
+            {
+                method: "POST",
+                headers: {
+                    "Content-Type": "application/json",
+                },
+                body: JSON.stringify(this.state),
+            }
+        );
+        let result = await promise.json();
+        this.resultHandler(result);
+    }
+    resultHandler(result) {
+        if (result.error) {
+            alert("Wrong info");
+        } else {
+            this.props.isAuthed()
+            this.props.history.push('/')
```

Comment:
This now does both of the things we wanted, invoked isAuthed() that is bound to the app and redirects us !

Reply:
Awesome!!

it looks like the .then that was attached to the async await method was edited out too (or not sure if that was while we were paired)